### DISCUSSION TITLE
Update data was corrupted

### DISCUSSION THREAD

**Discussion by chrisinhim** [audacity/audacity#4245] [Q&A]
### When I clicked on Help -> "Check for Updates..." I received the below error:

![image](https://user-images.githubusercontent.com/43258549/215156850-9c760d67-223b-4e40-9d12-0163b0273daf.png)

{
    "timestamp": 1674841222,
    "event_id": "a4a91a81dbb7ea47871a87beea8d246a",
    "platform": "native",
    "release": "audacity@3.2.1",
    "contexts": {
        "os": {
            "type": "os",
            "name": "Windows",
            "version": "10.0.22621"
        }
    },
    "exception": {
        "values": [
            {
                "type": "Error_checking_for_update",
                "value": "Update data was corrupted.",
                "mechanism": {
                    "type": "runtime_error",
                    "handled": false
                }
            }
        ]
    }
}

### So I clicked on here link on the error pop up and got this:
![image](https://user-images.githubusercontent.com/43258549/215157575-7188ce65-e320-4cba-bc6e-d69a70fce358.png)

I got an error when visiting audacityteam.org/errors.

Error code: 1020
Ray ID: 79034cba7c9b2e96
Country: IN
Data center: hyd01
IP: 49.205.236.70
Timestamp: 2023-01-27 17:45:01 UTC

### Steps to reproduce

1. Audacity app
2. Click on Help - Check for Updates...
3. Click on the "here" link given in the error pop up.

### Expected behavior

When I click on Help -> Check for Updates..., I should get information whether my Audacity software is up to date with the latest update or if there is any new version available.

### Audacity Version

other

### Operating system

Windows

### Additional context

Looks like Cloudflare messed up something with respect to their firewall security settings which are treating my computer as insecure. I don't know what they are. My computer is safe as per my knowledge. If you think I'm missing something with respect to my computer's safety, please help me identify that.

**Comment by LWinterberg** [comment]
Seems like your IP is blacklisted for some reason. I'll look into it on monday, in the meantime, you can download the latest version here: https://github.com/audacity/audacity/releases/latest

**Reply by TheSiZaGaming** [reply]
> Seems like your IP is blacklisted for some reason. I'll look into it on monday, in the meantime, you can download the latest version here: https://github.com/audacity/audacity/releases/latest

I have the same issue, and am also in India. Does this have something to do with the location?

**Reply by TheSiZaGaming** [reply]
I think I figured it out. It's definitely got to do with the region. I connected to a United States VPN and was able to update. Weird.

**Reply by rajdeepbaruah** [reply]
I thought only I had this problem since no one was talking about this issue or found any sort of video. Finally found you all.

**Reply by TheSiZaGaming** [reply]
Wonder why so many FOSS programs have issues in India. VLC's website stopped working a couple of months ago (now fixed) and Audacity still doesn't update. Super strange

**Reply by chrisinhim** [reply]
Today I clicked on the download button to download ffmpeg codec, but again got blocked as follows:
![image](https://github.com/audacity/audacity/assets/43258549/1c80368b-08c6-434e-ba05-2d921fa38bc8)

Here the issue is Cloudflare is considering my computer to be a threat. I remember some other websites also used Cloudflare and they didn't have this issue. I don't understand why audacityteam.org alone has this issue.

![image](https://github.com/audacity/audacity/assets/43258549/3c70fbcc-b0b7-480b-b5b1-4b07e399e707)

Is anybody working on this?
